2.0/m_geoipban.cpp

```diff
@@ -40,10 +40,23 @@ class ModuleGeoIPBan : public Module
 {
 	LocalStringExt ext;
 	GeoIP* gi;
-
+	GeoIP* fi;
+	
 	std::string* SetExt(User* user)
 	{
-		const char* c = GeoIP_country_code_by_addr(gi, user->GetIPString());
+		char buf[16];
+		const char* c = "UNK";
+		
+		if (inet_pton(AF_INET, user->GetIPString(), buf)) {
```

Comment:
You can check if `client_sa.sa.sa_family` is `AF_INET` or `AF_INET6` instead of doing this.

Reply:
Are you sure user->client_sa.sa.sa_family is available in insp20 ?

Reply:
Yes.